pynamodb/tests/integration/test_transaction_integration.py

```diff
@@ -0,0 +1,254 @@
+import os
+import uuid
+from datetime import datetime
+
+import pytest
+
+from pynamodb.connection import Connection
+from pynamodb.exceptions import PutError, DoesNotExist
+
+from pynamodb.attributes import NumberAttribute, UnicodeAttribute, UTCDateTimeAttribute, BooleanAttribute
+from pynamodb.connection.transactions import TransactGet, TransactWrite
+
+from pynamodb.models import Model
+
+IDEMPOTENT_PARAMETER_MISMATCH = 'IdempotentParameterMismatchException'
+PROVISIONED_THROUGHPUT_EXCEEDED = 'ProvisionedThroughputExceededException'
+RESOURCE_NOT_FOUND = 'ResourceNotFoundException'
+TRANSACTION_CANCELLED = 'TransactionCanceledException'
+TRANSACTION_IN_PROGRESS = 'TransactionInProgressException'
+
+
+class User(Model):
+    class Meta:
+        region = 'us-east-1'
+        table_name = 'user'
+
+    user_id = NumberAttribute(hash_key=True)
+
+
+class BankStatement(Model):
+
+    class Meta:
+        region = 'us-east-1'
+        table_name = 'statement'
+
+    user_id = NumberAttribute(hash_key=True)
+    balance = NumberAttribute(default=0)
+    active = BooleanAttribute(default=True)
+
+
+class LineItem(Model):
+
+    class Meta:
+        region = 'us-east-1'
+        table_name = 'line-item'
+
+    user_id = NumberAttribute(hash_key=True)
+    created_at = UTCDateTimeAttribute(range_key=True, default=datetime.now())
+    amount = NumberAttribute()
+    currency = UnicodeAttribute()
+
+
+class DifferentRegion(Model):
+
+    class Meta:
+        region = 'us-east-2'
+        table_name = 'different-region'
+
+    entry_index = NumberAttribute(hash_key=True)
+
+
+TEST_MODELS = [
+    BankStatement,
+    DifferentRegion,
+    LineItem,
+    User,
+]
+
+
+CONNECTION = None
+
+
+def _get_connection(url):
```

Comment:
```
@pytest.fixture(scope='module')
def connection(dburl):
    return Connection(host=url)
```

Reply:
`ddb_url` is scoped on the function level, so it can't be used with a `module` level fixture, which is why i made the helper :/

Reply:
nvm, can just update its fixture

Reply:
Maybe change `ddb_url` to be `session='scope'`? It doesn't really change.

Reply:
ye, that's what i did